packages/reporter/cypress/integration/unit/runnables_store_spec.ts

```diff
@@ -259,5 +275,12 @@ describe('runnables store', () => {
       instance.reset()
       expect(instance.testById('1')).to.be.undefined
     })
+
+    it('resets runnablesHistory', () => {
+      instance.setRunnables({ tests: [createTest('1')] })
+      instance.setRunningSpec('previous')
+      instance.reset()
+      expect(Object.keys(instance.runnablesHistory).length).to.equal(0)
```

Comment:
https://www.chaijs.com/api/bdd/#method_empty

```suggestion
      expect(instance.runnablesHistory).to.be.empty
```